lib/github_changelog_generator/generator/generator.rb

```diff
@@ -151,5 +148,25 @@ def fetch_issues_and_pr
       add_first_occurring_tag_to_prs(@sorted_tags, @pull_requests)
       nil
     end
+
+    # Remove the previously assigned fixed message.
+    # @param [String] log Old lines are mixed
```

Comment:
```suggestion
    # @param log [String] Old lines are fixed
```

- a typo 🙂 
- newer YARD style: naming the parameter before the type description